lib/translation_maps/locations.rb

```diff
@@ -1,323 +1,325 @@
 {"anxa"=>"Forrestal Annex",
```

Comment:
I had created this manifest as a backup in case bibdata is down (or what used to occur more frequently, when I was off the campus network). The list gets updated and at the start of each index process. The translation map utility requires a file to read from.